packages/syntax-core/src/Button/Button.stories.tsx

```diff
@@ -0,0 +1,78 @@
+import { StoryObj, Meta } from "@storybook/react";
+import Button from "./Button";
+import { Color, Size } from "../constants";
+
+export default {
+  title: "Button",
+  component: Button,
+  parameters: {
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/p7LKna9JMU0JEkcKamzs53/Cambly-Design-System-(Draft)?node-id=994%3A2346",
+    },
+  },
+  argTypes: {
+    color: {
+      options: [
+        Color.PRIMARY,
+        Color.SECONDARY,
+        Color.TERTIARY,
+        Color.DESTRUCTIVE_PRIMARY,
+        Color.DESTRUCTIVE_SECONDARY,
+        Color.DESTRUCTIVE_TERTIARY,
+        Color.SUCCESS,
+        Color.BRANDED,
+      ],
+      control: { type: "radio" },
+    },
+    size: {
+      options: [Size.SMALL, Size.MEDIUM, Size.LARGE],
+      control: { type: "radio" },
+    },
+    disabled: {
+      control: "boolean",
+    },
+    loading: {
+      control: "boolean",
+    },
+    fullWidth: {
+      control: "boolean",
+    },
+    onClick: { action: "clicked" },
+  },
+  tags: ["autodocs"],
+} as Meta<typeof Button>;
+
+export const Default: StoryObj<typeof Button> = {
```

Comment:
should there be a story using end/start icon?

Reply:
@somethiiing Yes, updated